src/extras/core/Interpolations.d.ts

```diff
@@ -0,0 +1,5 @@
+export namespace Interpolations {
```

Comment:
I don't think it's not necessary to export a namespace since all three function are directly exported. AFAIK, a namespace does only make sense if `Interpolations.js` would have a similar structure like `Math.js`.

Reply:
I saw this code before tried to code this PR.

https://github.com/mrdoob/three.js/blob/d5743a467685b27bf98c80fe41d5e2890b7e5848/src/extras/curves/CatmullRomCurve3.d.ts#L5-L6

l look `math.d.ts` also has a namespace.

https://github.com/mrdoob/three.js/blob/d5743a467685b27bf98c80fe41d5e2890b7e5848/src/math/Math.d.ts#L5-L6

You mean, I should delete `export namespace CurveUtils { }`, but this code does't work.
like: 

```js
export function CatmullRom(t: number, p0: number, p1: number, p2: number, p3: number): number;
export function QuadraticBezier(t: number, p0: number, p1: number, p2: number): number;
export function CubicBezier(t: number, p0: number, p1: number, p2: number, p3: number): number;
```

TBH, I don't get you point 100%, so I don't understand how to do, please share you code.

Reply:
Never mind! I initially thought a namespace would not be necessary. But it seems it is required. Sry, for the confusion.